addons/account_peppol/models/res_partner.py

```diff
@@ -63,10 +89,17 @@ def _check_peppol_participant_exists(self, edi_identification):
         try:
             response = requests.get(smp_url, timeout=TIMEOUT)
         except requests.exceptions.ConnectionError:
-            return False
+            return None  # returning None to be consistent with etree
```

Comment:
below, I need to check whether the participant exists
`if participant_info := self._get_participant_info(...)`
and you have to check that by specifying `is not None` (otherwise you get a deprecation warning from lxml)
I think it's easier to then make sure we return None and not False for that condition

Reply:
alright, I updated it

Reply:
could you please take a look again? :pray: @xmo-odoo